### PR TITLE
DOCS-2544: Add backlinks to docs

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

**Review by njooma — APPROVED** [review]

File: lib/src/components/camera/camera.dart [hunk 1]
```diff
@@ -16,28 +18,41 @@ abstract class Camera extends Resource {
   /// ```
   /// var nextImage = await myCamera.image();
   /// ```
+  ///
+  /// For more information, see [Camera component](https://docs.viam.com/components/camera/).
+
```
Comment (njooma) [inline lib/src/components/camera/camera.dart:23]: extraneous new line

File: lib/src/app/app.dart [hunk 2]
```diff
@@ -12,61 +12,77 @@ typedef RobotPartLogPage = GetRobotPartLogsResponse;
...
+  ///
+  /// For more information, see [Fleet Management API](https://docs.viam.com/appendix/apis/fleet/).
   Future<String> getUserIdByEmail(String email) async {
     final request = GetUserIDByEmailRequest()..email = email;
     final GetUserIDByEmailResponse response = await _client.getUserIDByEmail(request);
     return response.userId;
   }
 
   /// Create a new [Organization]
-  Future<Organization> createOrganization(String name) async {
```
Comment (njooma) [inline lib/src/app/app.dart:28]: Looks like this was removed by accident?

File: lib/src/app/app.dart [hunk 3]
```diff
@@ -12,61 +12,77 @@ typedef RobotPartLogPage = GetRobotPartLogsResponse;
...
+  ///
+  /// For more information, see [Fleet Management API](https://docs.viam.com/appendix/apis/fleet/).
   Future<Organization> getOrganization(String organizationId) async {
     final getOrganizationRequest = GetOrganizationRequest()..organizationId = organizationId;
     final GetOrganizationResponse response = await _client.getOrganization(getOrganizationRequest);
     return response.organization;
   }
 
   /// Checks for namespace availablity throughout all [Organization]s.
-  Future<bool> getOrganizationNamespaceAvailability(String publicNamespace) async {
```
Comment (njooma) [inline lib/src/app/app.dart:63]: And this was also removed by accident?

**Review by njooma — CHANGES_REQUESTED** [review]

File: lib/src/app/app.dart (hunk 2 above)
Comment (npentrel) [reply lib/src/app/app.dart:28]: yep - sorry

File: lib/src/app/app.dart (hunk 3 above)
Comment (npentrel) [reply lib/src/app/app.dart:63]: yep

File: lib/src/components/servo/servo.dart [hunk 4]
```diff
@@ -3,34 +3,58 @@ import '../../resource/base.dart';
...
+  /// Get the [ResourceName] for this [Servo] with the given [name]
+  ///
+  /// For more information, see [Servo component](https://docs.viam.com/components/servo/).
+  ///
   /// ```
   /// await myServo.stop();
   /// ```
   Future<void> stop({Map<String, dynamic>? extra});
 
   /// Whether the [Servo] is currently moving.
```
Comment (npentrel) [inline lib/src/components/servo/servo.dart:45]:
```suggestion
  /// Get if the [Servo] is currently moving.
```

File: lib/src/components/servo/servo.dart [hunk 5]
```diff
@@ -3,34 +3,58 @@ import '../../resource/base.dart';
...
+
+  /// Get if the [Servo] is currently moving.
+  ///
+  /// For more information, see [Servo component](https://docs.viam.com/components/servo/).
+  Future<bool> isMoving();
+
+  /// Get the [ResourceName] for this [Servo] with the given [name]
+  ///
+  /// For more information, see [Servo component](https://docs.viam.com/components/servo/).
+  ///
```
Comment (npentrel) [inline lib/src/components/servo/servo.dart:39]:
```suggestion
```

File: lib/src/components/servo/servo.dart [hunk 6]
```diff
@@ -3,34 +3,58 @@ import '../../resource/base.dart';
...
   /// ```
   /// var angle = await myServo.position();
   /// ```
   Future<int> position({Map<String, dynamic>? extra});
 
   /// Stop the [Servo]. It is assumed that the servo stops immediately.
   ///
+  /// For more information, see [Servo component](https://docs.viam.com/components/servo/).
+  ///
+  Future<void> stop({Map<String, dynamic>? extra});
```
Comment (npentrel) [inline lib/src/components/servo/servo.dart:29]:
```suggestion
```

File: lib/src/components/arm/arm.dart [hunk 7]
```diff
@@ -3,15 +3,19 @@ import '../../resource/base.dart';
...
+///
+/// For more information, see [Arm component](https://docs.viam.com/components/arm/).
 abstract class Arm extends Resource {
   static const Subtype subtype = Subtype(resourceNamespaceRDK, resourceTypeComponent, 'arm');
 
   /// Get the current [Pose] of the end of the arm.
   ///
   /// ```
-  /// // Get the pose of an arm named "myArm"
+  /// // Get the pose of an arm named "myArm2"
```
Comment (npentrel) [inline lib/src/components/arm/arm.dart:14]:
```suggestion
```

File: lib/src/components/motor/motor.dart [hunk 8]
```diff
@@ -110,22 +131,30 @@ abstract class Motor extends Resource {
...
+  /// For more information, see [Motor component](https://docs.viam.com/components/motor/).
+  ///
   /// ```
   /// // Check whether the motor is moving.
   /// var moving = await myMotor.isMoving();
   /// ```
   Future<bool> isMoving({Map<String, dynamic>? extra});
 
-  /// Get the [ResourceName] for this [Motor] with the given [name].
+  /// Get the [ResourceName] for this [Motor] with the given [name]
```
Comment (npentrel) [inline lib/src/components/motor/motor.dart:142]:
```suggestion
  /// Get the [ResourceName] for this [Motor] with the given [name].
```

**Comment by npentrel** [comment]
@njooma Thanks for the review. I've made the changes and merged in updates.

**Review by njooma — APPROVED** [review]

**Pull request merged**